webpack.config.cjs

```diff
@@ -168,6 +168,10 @@ const sharedConfig = {
   ].filter(Boolean),
   optimization: {
     sideEffects: true,
+    splitChunks: {
```

Comment:
Let's also make sure to add this to `activationNotice` below; or, better, simply remove the special `splitChunks` config for `activationNotice`

Reply:
I couldn't for the life of me work out why this is needed. It doesn't exact any css as it uses styled components. Do you know why this was originally built like this?

Reply:
Probably some copy-paste. Let's remove the custom `optimization` configuration from `activationNotice`.

Reply:
Removed.